packages/editor/src/lib/editor/managers/LicenseManager.ts

```diff
@@ -213,15 +236,20 @@ export class LicenseManager {
 				`Please reach out to ${LICENSE_EMAIL} if you would like to use tldraw on other domains.`,
 			])
 		}
+		if (result.license.flags > HIGHEST_FLAG * 2) {
```

Comment:
i might be being dense here 😆 
what is the * 2 for?

Reply:
Oh, I should have added the comment. If we add a new flag it will always be 2 times the current highest flag. And the sum of all the existing flags is `2 * HIGHEST_FLAG - 1`, so just lower than `HIGHEST_FLAG * 2`.

Reply:
if the highest now is 4 and we do * 2, wouldn't the check fail?
8 > 4 * 2 would just be equal, eh?

i think actually, correct me if i'm wrong, we can just do `flags > HIGHEST_FLAG`, no?

so, 0x10 (a flag higher than expected) > 0x0a (highest)
or, in binary, 0b10000 > 0b01000

Reply:
oh i see, gotcha, now it makes sense 👍